src/vs/workbench/browser/parts/editor/editorDropTarget.ts

```diff
@@ -334,6 +334,8 @@ class DropOverlay extends Themable {
 								editor: draggedEditor.identifier.editor,
 								options: fillActiveEditorViewState(sourceGroup, draggedEditor.identifier.editor, {
 									pinned: true,											// always pin dropped editor
+									// TODO review: this means dragging a combination of sticky and unsticky
```

Comment:
See comment

Reply:
Isn't this correct? All editors should be sticky after dopping or all unsticky. It would be very weird to have a mixed scenario